xml/security_preface.xml

```diff
@@ -8,753 +8,450 @@
+   Vulnerabilities are assigned a <emphasis>Common Vulnerabilities
+   and Exposures</emphasis> (<emphasis>CVE</emphasis>) number and a
+   <emphasis>Common Vulnerability Scoring System</emphasis>
+   (<emphasis>CVSS</emphasis>) score. The score helps identifying
+   the severity of vulnerabilities.
+  </para>
+  <para>
+   &suse; provides a feed of security advisories. It is available at
+   <link xlink:href="https://www.suse.com/en-us/support/update/" />.
+   There is also a list of security updates by CVE number available at
+   <link xlink:href="https://www.suse.com/en-us/security/cve/" />.
+  </para>
+  <para>
+   In general, administrators should be prepared for severe
+   vulnerabilities in their systems. This includes hardening all
+   computers as far as possible. Also, we recommend to have predefined
+   procedures in place for quickly installing updates for severe
+   vulnerabilities.
+  </para>
+  <para>
+   To reduce the damage of possible attacks, use restrictive file
+   permissions. See <xref linkend= "sec.security.acls.traditional" />.
+   &suse; provides a guide to harden &productname;.<phrase
+   os="sles">For details, see <xref
+   linkend="book.hardening"/>.</phrase>
+  </para>
+  <para>
+   Other useful links:
+  </para>
+  <itemizedlist>
    <listitem>
     <para>
-     Get and install the updated packages recommended by security
-     announcements as quickly as possible.
+     <link
+     xlink:href="http://lists.opensuse.org/opensuse-security-announce/"/>
+     , mailing list with &opensuse; security announcements.
     </para>
    </listitem>
    <listitem>
-    <para>
-     Stay informed about the latest security issues:
-    </para>
-    <itemizedlist mark="bullet" spacing="normal">
-     <listitem>
-      <para>
-       <link xlink:href="http://lists.opensuse.org/opensuse-security-announce/"/>
-       is the &suse; mailing list for security announcements. It is a
-       first-hand source of information regarding updated packages and
-       includes members of &suse;'s security team among its active
-       contributors. You can subscribe to this list on page
-       <link xlink:href="http://en.opensuse.org/openSUSE:Mailing_lists"/>.
-      </para>
-     </listitem>
-     <listitem>
-      <para>
-       Find &suse; security advisories at
-       <link os="sles;sled" xlink:href="http://www.suse.com/support/update/"/><link os="osuse" xlink:href="https://www.suse.com//security/cve/"/>.
-      </para>
-     </listitem>
-     <listitem>
-      <para>
-       <literal>bugtraq@securityfocus.com</literal> is one of the best-known
-       security mailing lists worldwide. Reading this list, which receives
-       between 15 and 20 postings per day, is recommended. More information
-       can be found at <link xlink:href="http://www.securityfocus.com"/>.
-      </para>
-     </listitem>
-    </itemizedlist>
+    <para><link
+     xlink:href="https://nvd.nist.gov/home"
+    />, the National Vulnerability Database</para>
```

Comment:
Not really wrong but line breaks look weird, i.e. no break after para and before /para but breaks within the link tag. (Same below.)